uber/config.py

```diff
@@ -315,6 +315,12 @@ def _is_intstr(s):
         return s[1:].isdigit()
     return s.isdigit()
 
+if not c.KIOSK_CC_ENABLED:
+    del _config['enums']['door_payment_method']['stripe']
```

Comment:
I personally would prefer to have ``c.STRIPE`` and ``c.GROUP`` still defined, but have all other aspects of the enums gone (e.g. not anywhere in ``_OPTS`` or ``_VARS``, etc).  That way I can still write one-off scripts which e.g. refer to ``c.STRIPE`` when performing queries and such.

The best way to do this is probably to keep the code exactly how you've got it now here in ``config.py`` but in each of the ``if`` blocks add a line which defines the missing option.  The code which currently defines and sets those values is at
https://github.com/magfest/ubersystem/blob/212fd179b18a665d079843ffed9beed349864e78/uber/config.py#L68

```python
val = int(sha512(name.upper().encode()).hexdigest()[:7], 16)
setattr(self, name.upper(),  val)
```

What we could do is move that out into its own function in ``c`` like

```python
    def create_enum_val(self, name):
        val = int(sha512(name.upper().encode()).hexdigest()[:7], 16)
        setattr(self, name.upper(), val)
        return val
```

That would allow us to define these kinds of enum options on ``c`` even as we remove them, e.g. when removing the group option, we could say

```python
if not c.GROUPS_ENABLED:
    del _config['enums']['door_payment_method']['group']
    c.create_enum_val('group')
```

Something along those lines would be my preferred approach.